src/routes/data-router.ts

```diff
@@ -0,0 +1,69 @@
+import { Router } from "express";
+import { getRepository } from "typeorm";
+import { Todo } from "../entities/todo";
+
+const dataRouter = Router();
+
+dataRouter.get("/Tasks", async (req, res) => {
```

Comment:
כבר הגדרת שה router הזה מקשיב ל Tasks/ למעלה ב api router
כרגע בשביל להגיע לפה צריך לשלוח בקשה ל Tasks/Tasks/
```suggestion
dataRouter.get("/", async (req, res) => {
```

Reply:
טופל, גם בpost